src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+private Random random = new Random();
     public String getRandomColor() {
```

Comment:
Please add an empty line between these two sections and align your field correctly.